internal/setup/setup.go

```diff
@@ -203,5 +210,20 @@ func SetupWith(ctx context.Context, config interface{}, l envconfig.Lookuper) (*
 		}
 	}
 
+	// Configure and initialize the observability exporter.
```

Comment:
This should probably happen after the secret manager and kms setup (before database)